build.gradle.kts

```diff
@@ -19,33 +19,45 @@ tasks.withType<Test> {
 
 repositories {
     mavenCentral()
-    maven(url = "http://packages.confluent.io/maven/")
+    maven(url = "https://packages.confluent.io/maven/")
 }
 
 configure<JavaPluginConvention> {
     sourceCompatibility = JavaVersion.VERSION_11
     targetCompatibility = JavaVersion.VERSION_11
 }
 
+sourceSets {
```

Comment:
Why do we need this?

Reply:
It adds the .avsc to the published jar. This allows us to use the schemas in other projects as dependencies. I will add a comment